DbTables/inc/CalCosmicT0Align.hh

```diff
@@ -0,0 +1,70 @@
+#ifndef DbTables_CalCosmicT0Align_hh
+#define DbTables_CalCosmicT0Align_hh
+
+
+#include <string>
+#include <iomanip>
+#include <sstream>
+#include <map>
+#include "Offline/DbTables/inc/DbTable.hh"
+#include "Offline/DataProducts/inc/CaloSiPMId.hh"
+
+namespace mu2e {
+
+  class CalCosmicT0Align : public DbTable {
+  public:
+
+    class Row {
+    public:
+      Row(CaloSiPMId  roid, float Tcorr, float Terr, float chisq):_roid(roid),_Tcorr(Tcorr), _Terr(Terr), _chisq(chisq){}
+      CaloSiPMId   roid() const { return _roid;}
+      float Tcorr() const { return _Tcorr; }
+      float Terr() const { return _Terr; }
+      float chisq() const { return _chisq; }
+
+    private:
+      CaloSiPMId   _roid;
+      float _Tcorr;
+      float _Terr;
+      float _chisq;
+    };
+
+    constexpr static const char* cxname = "CalCosmicT0Align";
+
+    CalCosmicT0Align():DbTable(cxname,"cal.cosmicT0align","roid,Tcorr,Terr,chisq"){}
+
+    const Row& row(std::uint16_t roid) const { return _rows.at(roid); }
```

Comment:
In the other tables, we had a CaloSIPMId here

Reply:
I dont see that in other tables, but maybe it needs to be implemented for all?

Reply:
I was comparing to CalEnergyCalib, which uses it, but it looks like that is the only table which does.  But I think this is part of the general comment, how much do you want to embrace CaloSiPMID, and how much right now?  I won't insist in this PR, but eventually there should be a consistent pattern.   I also see some tables use int and some uint16_t